pom.xml

```diff
@@ -53,11 +53,8 @@ under the License.
 
   <modules>
     <module>flink-ml-api</module>
-    <module>flink-ml-lib</module>
```

Comment:
We should probably add flink-ml-lib back.

Reply:
Compilation will fail after I add this module back in the PR. I will ask Yun to merge https://github.com/apache/flink-ml/pull/19 first.